<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>元对象</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="boy">
       <property name="text">
        <string>设置男生年龄</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QSpinBox" name="boySpinBox">
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="addBoyAge">
       <property name="text">
        <string>男生加一</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>清空文本</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QPushButton" name="showMeta">
       <property name="text">
        <string>显示元信息</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="girl">
       <property name="text">
        <string>设置女生年龄</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="girlSpinBox">
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="addGirlAge">
       <property name="text">
        <string>女生加一</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
       <item>
        <widget class="QPushButton" name="qMessage">
         <property name="text">
          <string>测试按钮01</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qFile">
         <property name="text">
          <string>测试按钮02</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qFont">
         <property name="text">
          <string>测试按钮03</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="qInput">
         <property name="text">
          <string>测试按钮04</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
